Rubberduck.Parsing/Annotations/Concrete/DefaultMemberAnnotation.cs

```diff
@@ -11,8 +12,8 @@ namespace Rubberduck.Parsing.Annotations
     public sealed class DefaultMemberAnnotation : FixedAttributeValueAnnotationBase
     {
         public DefaultMemberAnnotation()
-            : base("DefaultMember", AnnotationTarget.Member, "VB_UserMemId", new[] { "0" })
+            : base("DefaultMember", AnnotationTarget.Member, "VB_UserMemId", new[] { WellKnownDispIds.Value })
```

Comment:
Would having a `WellKnownDispIds.Default` be awkward?

Reply:
I would imagine only for someone who has memorized the DISPID constants off of which you based `WellKnownDispIds`? The thought of changing definitely crossed my mind as well.

Reply:
IDK maybe `VALUE` is what the official docs call it, and if that's the case then so be it. Otherwise, ...where is it used? If this is the only usage, then `Default` or even `DefaultMember` would be a better name. If it's used elsewhere with another meaning, it's probably best to just leave it alone.

Reply:
Is it too messy to have both members in `WellKnownDispIds`? I tried to actually clone RD to answer the question about where it is used but VS is having problems with NuGet.

Reply:
Update: I think I got it to work and from what I can tell it is not used anywhere else. Also those changes I made are wrong because it expects a string and VBA instills some less than regimented habits when it comes to using integers and strings...

Reply:
@retailcoder, 
> Would having a WellKnownDispIds.Default be awkward? 

FWIW, [`MS-OAUT`](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-oaut/cb9d0131-c6bd-463d-9c40-7264856a10c5) has it listed as `DISPID_VALUE`. 😕

Reply:
@bclothier yeah that's exactly where I suspected the name came from. Plus the xmldocs, I think `Value` is just what it needs to be named to keep consistent with the docs, ...just unfortunate that it needs xmldocs to make sense. The `DISPID_Xxxxxx` identifiers should probably be mentioned in the xmldoc-summary comment.